test/tests/functional/pbs_soft_walltime.py

```diff
@@ -687,3 +711,323 @@ def test_preempt_order(self):
 
         self.server.expect(JOB, {ATTR_state: 'R'}, id=jid3)
         self.server.expect(JOB, {ATTR_state: 'S'}, id=jid1)
+
+    def test_soft_values_default(self):
```

Comment:
if soft walltime only truly supports int and long type values, then should we also test for floating point values?

Reply:
updated ..